src/jit/lsra.cpp

```diff
@@ -8536,6 +8545,13 @@ void LinearScan::recordMaxSpill()
         maxSpill[TYP_FLOAT] += 1;
     }
 #endif // _TARGET_X86_
+
+    if (compiler->opts.compUseSoftFP)
+    {
+        JITDUMP("Adding a spill temp for moving target address to a register.\n");
+        maxSpill[TYP_INT] += 1;
```

Comment:
What I stated here https://github.com/dotnet/coreclr/pull/13023#issuecomment-319255219 was strange part while LSRA, the C# code of this part is roughly like:

```cs
func(a, b);
// static SomeReturnType func(double, double)
```

While setting r0-r3 arguments, they are not fixed until the call. So `r0` is spilled (since no other free registers) for call address. Eventually the call address goes to `r8` though. And this probably requires one more temp variable. 
I think this could be an issue for general arm, not only armel, since x86/x64 usually does not store call address to a register, directly uses a full immediate address.

% The exact TC I was looking at is `JIT/Methodical/NaN/r8NaNmul_cs_do/r8NaNmul_cs_do.exe`.